hugolib/page_test.go

```diff
@@ -1536,7 +1537,8 @@ Content.
 }
 
 func TestShouldBuild(t *testing.T) {
-	t.Parallel()
+	htime.NowOverride = time.Date(2021, 11, 17, 20, 34, 58, 651387237, time.UTC)
```

Comment:
Please don't change existing tests; all of Hugo when this flag is not set should work with the system clock, so it's vital that we keep existing tests as is to verify this. If you have new test cases, create a new test func.